src/main/java/uk/gov/companieshouse/web/accounts/service/smallfull/SmallFullService.java

```diff
@@ -14,6 +14,15 @@
      */
     void createSmallFullAccounts(String transactionId, String companyAccountsId) throws ServiceException;
 
+    /**
+     * Update a small full resource
+     * @param smallFullApi The updated resource you want to submit
+     * @param transactionId The id of the CHS transaction
+     * @param companyAccountsId The company accounts identifier
+     * @throws ServiceException on creation failure
+     */
+    void updateSmallFullAccounts(SmallFullApi smallFullApi, String transactionId, String companyAccountsId) throws ServiceException;
```

Comment:
No major preference, but we could change this method signature to take a `LocalDate periodEndOn` rather than a `SmallFullApi`. At the moment, when used above in the controller we are submitting a lot of read only data to the API. We don't necessarily need all of the data in the request body, just the period end on. So within the implementation of this method we could construct a new `SmallFullApi` object and if `periodEndOn` were not null, set that on the new instance of `SmallFullApi`?

Reply:
This is now implemented